polar-core/src/vm.rs

```diff
@@ -1490,6 +1523,15 @@ impl PolarVirtualMachine {
                 ])?;
             }
             Operator::Cut => {
+                if self.query_contains_partial {
+                    return Err(self.set_error_context(
+                        &term,
+                        error::RuntimeError::Unsupported {
+                            msg: "cannot use cut with a partial".to_string(),
```

Comment:
```suggestion
                            msg: "cannot use cut with a query containing a partial".to_string(),
```

Reply:
Is this necessary? The only time users will encounter this error is when they query with a partial for a rule containing a `cut`.

Reply:
this is fine nvm.